frontend/packages/ceph-storage-plugin/integration-tests/views/ocp-dashboard-card-healthcheck.view.ts

```diff
@@ -0,0 +1,34 @@
+import { execSync } from 'child_process';
+import { $, element, by } from 'protractor';
+import { getPodData } from '../utils/helpers';
+
+export const mainHealthCardStatus = $('.co-dashboard-card__body--top-margin.co-health-card__body');
+export const mainHealtGreenSvg = mainHealthCardStatus.$(
+  'svg[fill="var(--pf-chart-color-green-400)"]',
+);
+export const seeAllLink = element(
+  by.cssContainingText('.co-dashboard-card__button-link.btn.btn-link', 'See all'),
+);
+export const seeAllClose = $('button[aria-label="Close"]');
+export const seeAllCard = $('.pf-c-popover__body');
+export const seeAllCardSections = seeAllCard.$$('.co-health-card__item');
+export const seeAllStorageSection = seeAllCardSections.get(1);
+export const seeAllStorageGreenSvg = seeAllStorageSection.$(
+  'svg[fill="var(--pf-chart-color-green-400)"]',
+);
+export const mainHealtYellowSvg = mainHealthCardStatus.$('svg[fill="#f0ab00"]');
```

Comment:
Do we really need to use the hardcoded colors hex code? If yes, please create a constant.

Reply:
I don't think it has another another way to check status icon color (green, yellow and red) according to ocp status.
https://github.com/openshift/console/blob/2a4df9e82d79a40e4c69f7541226b98979d22631/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts#L51-L55